src/plugins/visualizations/common/constants.ts

```diff
@@ -26,3 +26,13 @@ export const VisualizeConstants = {
   EDIT_BY_VALUE_PATH: '/edit_by_value',
   APP_ID: 'visualize',
 };
+
+export const LegendSizes = {
+  AUTO: 0,
+  SMALL: 80,
+  MEDIUM: 130,
+  LARGE: 180,
+  EXTRA_LARGE: 230,
+} as const;
+
+export const DEFAULT_LEGEND_SIZE = LegendSizes.MEDIUM;
```

Comment:
This seemed like the right place to put these for usage in all the vis_types. Still open to input.

Reply:
If we have the legend sizes here, do we still need them in the Lens plugin? https://github.com/elastic/kibana/pull/130336/files#diff-d23e749e762688055d5cdbc52ed902a26a009743d66f901b1568728975bcbb9cR45